selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4CheckRunSuitesAndScopesTest.java

```diff
@@ -72,9 +63,10 @@ public void prepareTestProject() throws Exception {
         JUNIT4_PROJECT,
         ProjectTemplates.CONSOLE_JAVA_SIMPLE);
     ide.open(ws);
-    loader.waitOnClosed();
+    ide.waitOpenedWorkspaceIsReadyToUse();
+
     projectExplorer.waitItem(JUNIT4_PROJECT);
-    runCompileCommandByPallete(compileCommand);
+    runCompileCommand();
```

Comment:
Could you, please, explain, why verification the command palette had been removed?